packages/components/src/mobile/alt-text-settings/styles.scss

```diff
@@ -0,0 +1,29 @@
+.altTextEditor {
+	padding: 14px 0;
+	min-height: 80px;
+	max-height: 250px;
+}
+
+.altTextEditorDark {
+	color: $white;
+}
+
+.altTextFootnote {
+	font-size: $default-font-size;
+	color: $gray-text-min;
+	padding: 8px 16px 24px;
+}
+
+.altTextLink {
+	color: $blue-wordpress;
```

Comment:
I believe this is an old blue (which is not very clear, based on a quick glance at the code 😄 ). For reference, we typically use `blue-50` in light mode and `blue-30` in dark mode for the primary blue.

Reply:
Updated and noted for the future, thank you! Here are the updated screenshots of what this looks like in the Android app: 

| Light mode  | Dark mode |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/110747916-eebada80-8236-11eb-82d7-dc14d898f41b.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/110748048-1f9b0f80-8237-11eb-84e4-f70fd0449a8a.png" width="100%"> |

Reply:
Ah, that looks better! Thank you!